command-handler/src/commands/vm.js

```diff
@@ -1,5 +1,6 @@
 import hetzner from '../util/hetzner/hetzner-servers.js';
 import aws from '../util/aws/aws-server.js';
+import libvirt from '../util/libvirt/libvirt-server.js';
```

Comment:
would it make sense to call it the provisioner? since we will be calling our wrapper over libvirt?

Reply:
this might be a heavy lift. i see libvirt all over the place. So feel free to disregard